openedx/core/djangoapps/verified_track_content/management/commands/swap_from_auto_track_cohort_pilot.py

```diff
@@ -0,0 +1,269 @@
+            if not old_course_key:
+                raise CommandError("No old_course_key set for MigrateVerifiedTrackCohortsSetting with ID: '%s'"
+                                   % verified_track_cohorts_settings.id)
+
+            if not rerun_course_key:
+                raise CommandError("No rerun_course_key set for MigrateVerifiedTrackCohortsSetting with ID: '%s'"
+                                   % verified_track_cohorts_settings.id)
+
+            if not audit_cohort_names:
+                raise CommandError("No audit_cohort_names set for MigrateVerifiedTrackCohortsSetting with ID: '%s'"
+                                   % verified_track_cohorts_settings.id)
+
+            print "Running for MigrateVerifiedTrackCohortsSetting with ID='%s'" % verified_track_cohorts_setting.id
+
+            # Get the CourseUserGroup IDs for the audit course names from the old course
+            audit_course_user_group_ids = CourseUserGroup.objects.filter(
+                name__in=audit_cohort_names,
+                course_id=old_course_key,
+                group_type=CourseUserGroup.COHORT,
+            ).values_list('id', flat=True)
+
+            if not audit_course_user_group_ids:
+                raise CommandError(
+                    "No Audit CourseUserGroup found for course_id='%s' with group_type='%s' for names='%s'"
+                    % (old_course_key, CourseUserGroup.COHORT, audit_cohort_names)
+                )
+
+            # Get all of the audit CourseCohorts from the above IDs that are RANDOM
+            random_audit_course_user_group_ids = CourseCohort.objects.filter(
+                course_user_group_id__in=audit_course_user_group_ids,
+                assignment_type=CourseCohort.RANDOM
+            ).values_list('course_user_group_id', flat=True)
+
+            if not random_audit_course_user_group_ids:
+                raise CommandError(
+                    "No Audit CourseCohorts found for course_user_group_ids='%s' with assignment_type='%s"
+                    % (audit_course_user_group_ids, CourseCohort.RANDOM)
+                )
+
+            # Get the CourseUserGroupPartitionGroup for the above IDs, these contain the partition IDs and group IDs
+            # that are set for group_access inside of modulestore
+            random_audit_course_user_group_partition_groups = list(CourseUserGroupPartitionGroup.objects.filter(
+                course_user_group_id__in=random_audit_course_user_group_ids
+            ))
+
+            if not random_audit_course_user_group_partition_groups:
+                raise CommandError(
+                    "No Audit CourseUserGroupPartitionGroup found for course_user_group_ids='%s'"
+                    % random_audit_course_user_group_ids
+                )
+
+            # Get the single VerifiedTrackCohortedCourse for the old course
+            try:
+                verified_track_cohorted_course = VerifiedTrackCohortedCourse.objects.get(course_key=old_course_key)
```

Comment:
Needs to also check that the VerifiedTrackCohortedCourse instance is enabled. I disabled it on my test course, and the first failure I hit was on line 111.

Starting Swap from Auto Track Cohort Pilot command
Running for MigrateVerifiedTrackCohortsSetting with ID='2'
CommandError: No Verified CourseUserGroup found for course_id='course-v1:christina+1+1' with group_type='cohort' for names='[u'foo']'